### PR TITLE
alpine: update upstream location

### PR DESCRIPTION
**Pull request by jcvernaleo**
Closes: https://trac.macports.org/ticket/65686

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by jcvernaleo** [comment]
I was not about to build and test since I'm not at my mac today but this is just changing the upstream location for download, no code change.

**Review by mascguy — APPROVED** [review]
LGTM. @jerryyhom ?

File: mail/alpine/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                alpine
 version             2.25
-revision            2
+revision            3
```
Comment (reneeotten) [inline mail/alpine/Portfile:7]: I don't think there is a reason to increase the `revision` field here

File: mail/alpine/Portfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ long_description    Alpine is a tool for reading, sending, and \
                     managing electronic messages that was designed \
                     with novice users in mind.
 
-master_sites        http://alpine.x10host.com/alpine/release/src
+master_sites        http://alpineapp.email/alpine/release/src
```
Comment (reneeotten) [inline mail/alpine/Portfile:19]: this should ideally use `https`

**Comment by jerryyhom** [comment]
Ehh, @reneeotten is doubly correct; thank you!

File: mail/alpine/Portfile (hunk 2 above)
Comment (jcvernaleo) [reply mail/alpine/Portfile:19]: Well, that one is a bit embarrassing.  I totally didn't notice that.

File: mail/alpine/Portfile (hunk 1 above)
Comment (jcvernaleo) [reply mail/alpine/Portfile:7]: You are correct here as well.  Haven't done any macports or BSD ports files in a while so I forgot the rule on revision.

**Comment by jcvernaleo** [comment]
@jerryyhom @reneeotten fixed both problems (both of which I do normally know better about).

**Pull request merged**